simulations/vip-140/vip-140/simulation.ts

```diff
@@ -0,0 +1,199 @@
+
+const CHAINLINK_ORACLE = "0x1B2103441A0A108daD8848D8F5d790e4D402921F";
+const CHAINLINK_ORACLE_IMPL_OLD = "0xEebBfFC42e26386C631914e69B86aD398e91DD7f";
+const CHAINLINK_ORACLE_IMPL = "0x38120f83734F719dc199109e09A822a80CD26EAd";
+
+const BOUND_VALIDATOR = "0x6E332fF0bB52475304494E4AE5063c1051c7d735";
+const BOUND_VALIDATOR_IMPL_OLD = "0x753192648599F58eEEd782cBf9A5880fFfEfd133";
+const BOUND_VALIDATOR_IMPL = "0xCf0612CeafD63709d8f7EfE71EcD0aAbF075f6b1";
+
+const BINANCE_ORACLE = "0x594810b741d136f1960141C0d8Fb4a91bE78A820";
+const BINANCE_ORACLE_IMPL_OLD = "0xe38AbE42948ef249E84f4e935e4f56483C1EE3B9";
+const BINANCE_ORACLE_IMPL = "0x8bf46792022126aE7f3ac8F4914Ed66e7DEb7388";
+
+const TWAP_ORACLE = "0xea2f042e1A4f057EF8A5220e57733AD747ea8867";
+const TWAP_ORACLE_IMPL_OLD = "0x17D00a8926566CB4b02B25f4EE6ecC2cEB34A784";
+const TWAP_ORACLE_IMPL = "0x67c549A18AbfAd127b13F8d56738F43A21bB62A7";
+
+const PYTH_ORACLE = "0xb893E38162f55fb80B18Aa44da76FaDf8E9B2262";
+const PYTH_ORACLE_IMPL_OLD = "0x01e12AFa8D016D11dFBBde48e1a51038072b2129";
+const PYTH_ORACLE_IMPL = "0x1b8dE8fe17735B80E30e1bAbcD78A20F573a3e9e";
+
+const PROXY_ADMIN = "0x1BB765b741A5f3C2A338369DAb539385534E3343"
+const COMPTROLLER = "0xfd36e2c2a6789db23113685031d7f16329158384";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+const SD = "0x3bc5ac0dfdc871b365d159f728dd1b9a0b5481e8";
+
+const DUMMY_SIGNER = "0xF474Cf03ccEfF28aBc65C9cbaE594F725c80e12d";
+const MOCK_VTOKEN_CODE =
+  "608060405234801561001057600080fd5b506101c3806100206000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c806325671dcb1461003b5780636f307dc314610057575b600080fd5b610055600480360381019061005091906100f1565b610075565b005b61005f6100b8565b60405161006c9190610129565b60405180910390f35b806000806101000a81548173ffffffffffffffffffffffffffffffffffffffff021916908373ffffffffffffffffffffffffffffffffffffffff16021790555050565b60008054906101000a900473ffffffffffffffffffffffffffffffffffffffff1681565b6000813590506100eb81610176565b92915050565b60006020828403121561010357600080fd5b6000610111848285016100dc565b91505092915050565b61012381610144565b82525050565b600060208201905061013e600083018461011a565b92915050565b600061014f82610156565b9050919050565b600073ffffffffffffffffffffffffffffffffffffffff82169050919050565b61017f81610144565b811461018a57600080fd5b5056fea264697066735822122072c165598ea94093a05d15ef83a4a5cf715c200381a4687389a3455431698e7564736f6c63430008000033";
+const MOCK_VTOKEN_ABI = [
+  {
+    inputs: [{ internalType: "address", name: "_asset", type: "address" }],
+    name: "setUnderlyingAsset",
+    outputs: [],
+    stateMutability: "nonpayable",
+    type: "function",
+  },
+  {
+    inputs: [],
+    name: "underlying",
+    outputs: [{ internalType: "address", name: "", type: "address" }],
+    stateMutability: "view",
+    type: "function",
+  },
+];
+
+interface ILVTokenConfig {
+  name: string;
+  assetAddress: string;
+  price: string;
+}
+
+const ilPoolTokens: ILVTokenConfig[] = [
```

Comment:
I would add the rest of the assets configured in the Binance Oracle:
```
ANKR: 0xf307910A4c7bbc79691fD374889b36d8531B08e3
ankrBNB: 0x52F24a5e03aee338Da5fd9Df68D2b6FAe1178827
BTT: 0x352Cb5E19b12FC216548a2677bD0fce83BaE434B
FLOKI: 0xfb5B838b6cfEEdC2873aB27866079AC55363D37E
HAY: 0x0782b6d8c4551B9760e74c0545a9bCD90bdc41E5
```

Given we are already upgrading the Chainlink implementation, I would add a different list with the assets configured in Chainlink (in IL and Core pool). To confirm everything is working as expected